### DISCUSSION TITLE
PatchTool 7.0 7/12 更新翻譯

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#459] [Announcements]
- 修正部分指令碼造成的錯誤，感謝回報與協助。
- 更新部分翻譯，感謝 @BonBM 與 @Liteya 提供的翻譯。
- 文本內容量大，目前僅對一小部分內容（主要是與遊戲指令碼相關的部分）做了人工檢查。如果仍出現遊戲跳錯或崩潰，請盡可能紀錄案發現場和使用設定。

以下兩種方法擇一使用即可；請從release下方連結或內文連結下載檔案。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20240712.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20240712.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20240712.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- FFXIVChnTextPatch-20240628.jar：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- [backup-20240628.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.3.0.0/backup-20240628.rar)：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.3.0.3'>PatchTool 7.0 7/12 更新翻譯</a>.</em>

**Comment by Paul332300** [comment]
缺少 FFXIVChnTextPatch-20240712.rar 文件

**Comment by edward90595** [comment]
7/16版本更新後,用此Tool會讓Viper的蛇尾術相關技能無法按出.

**Comment by patrick0918** [comment]
請問有一些技能沒有翻譯,之後會更新嗎?
